src/genie/libs/parser/iosxe/c9500/show_platform.py

```diff
@@ -55,12 +55,18 @@ class ShowVersionSchema(MetaParser):
                     'model_rev_num': str,
                     'mb_rev_num': str,
                     'model_num': str,
-                    'system_sn': str,
+                    Optional ('system_sn'): {
+                        Any(): str,
+                    },
                     Optional('mem_size'): {
                         Any(): str,
                     },
-                    'license_level': str,
-                    'next_reload_license_level': str,
+                    Optional('license_level'): {
+                        Any(): str,
+                    },
+                    Optional('next_reload_license_level'): {
+                        Any(): str,
+                    },
```

Comment:
```suggestion
                    Optional('license_level'): str,
                    Optional('next_reload_license_level'): str,
```
what I meant on above comment is this. do we have a reason why we need to have `Any()` under these levels?

Reply:
confirmed